scripts/main_obs_prep.sh

```diff
@@ -1,8 +1,10 @@
-#!/bin/bash -l
+#!/bin/ksh
```

Comment:
without the -l it wont work on Hera

Reply:
> Modules with sh, bash, and ksh scripts
> Due to the way the POSIX standard is defined for bash, sh, and ksh you MUST add the -l option (that is a lowercase L) to the shebang (e.g. #!/bin/sh) line at the top of your script for all sh, bash, or ksh batch scripts. For example:
> 
> #!/bin/ksh -l
> 
> module load intel
> module load impi
> 
> srun -n 12 ​./xhpl


I think that the solution of "sh script.sh" and keeping the -l on the top of the script will work.

Reply:
@flampouris adding -l to ksh won't work at all.